docs/typescript/how-to-develop-a-workflow-definition-in-typescript.md

```diff
@@ -0,0 +1,37 @@
+---
+id: how-to-develop-a-workflow-definition-in-typescript
+title: How to develop a Workflow definition in Typescript
+sidebar_label: Develop a Workflow definition
+description: Develop a Workflow definition
+tags:
+  - developer-guide
+  - sdk
+  - typescript
+---
+
+A Workflow Function has two parts:
+
+- The function name is known as the Workflow Type.
+- The function implementation code (body) is known as the Workflow Definition.
```

Comment:
```suggestion
- The function name is the [Workflow Type](/docs/concepts/what-is-a-workflow-type/).
- The function implementation is the [Workflow Definition](/docs/concepts/what-is-a-workflow-definition).
```

"function implementation code" feels awk to me, and I'd say "body" is inaccurate, since I'd include the signature in the Definition.